doc/reference/guidelines.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight:: python
+
+=================
+Coding Guidelines
+=================
+
+Python
+======
+
+- PEP8 ignore E501
+- prefer % over format
```

Comment:
Yes that does work (with both `%` and `format`), but then we must mandate that translatable messages use keyword-based string formatting, never positional.